packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -617,8 +626,10 @@ describe("Garbage Collection Tests", () => {
                 clock.tick(1);
                 await updateAllNodesAndRunGC(garbageCollector);
                 const expectedEvents: Omit<ITelemetryBaseEvent, "category">[] = [];
-                if (deleteEventName) {
+                if (expectDeleteLogs) {
                     expectedEvents.push({ eventName: deleteEventName, timeout, id: nodes[3] });
+                } else {
+                    assert(!mockLogger.events.some((event) => event.eventName === deleteEventName), "Should not have any delete events logged");
```

Comment:
Why did we disable max-len?

Reply:
It's my personal preference - I do it in all test files I work in.  Often test code is more verbose - especially with assert message strings - and I like just letting the lines be long.

I take it you don't like long lines?

Reply:
Haha yes. It makes it harder to read the code on laptops. Also, the repo rule is to not have lines more than 80 characters :-). Anyway, I was just curious, not asking you to change it.

Reply:
I thought the repo rule was 120?

Reply:
My biggest beef is how it slows down the inner dev loop iterating on tests.  Maybe I will try to adjust my habit to disabling in the file only temporarily but not checking it in (cleaning up before merging).  Feel free to remove and fix up violations sometime.